pkg/connector/login.go

```diff
@@ -137,6 +141,70 @@ func (m *MetaCookieLogin) SubmitCookies(ctx context.Context, strCookies map[stri
 
 	FBID := m.getFBID(currentUser, initialTable)
 
+	// Store the cookies in the database
+
+	q := m.Main.store.GetSessionQuery()
+	session, err := q.GetByMetaID(ctx, FBID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get existing session: %w", err)
+	}
+	if session != nil {
+		log.Debug().Int64("fbid", FBID).Msg("Deleting existing session")
```

Comment:
Are there ever going to be multiple open sessions for the same FBID? Would only happen if sharing a Facebook/Instagram account, which would be weird so I guess not